torch/_inductor/decomposition.py

```diff
@@ -143,6 +143,23 @@ def convolution_backward(
     )
     return (grad_inp, grad_weight, grad_bias)
 
+@register_decomposition(aten.slice_scatter)
+def slice_scatter(self, src, dim=0, start=None, end=None, step=1):
+    if start == 0 and end >= 2**63 - 1 and step == 1:
+        return src
+    return NotImplemented
+
+@register_decomposition(aten.slice)
+def slice(self, dim=0, start=None, end=None, step=1):
+    if start == 0 and end >= 2**63 -1 and step == 1:
+        return self
```

Comment:
curiosity q: where does this happen in real-world code? And is this coming from code like `x[:] = ...`?

Reply:
It’s like, `x[:, :, index]` i think.